src/concurrent_log_handler/__init__.py

```diff
@@ -573,10 +575,28 @@ def _shouldRollover(self):
 
 
 class ConcurrentTimedRotatingFileHandler(TimedRotatingFileHandler, ConcurrentFileHandlerMixin):
-    pass
-
-
-# Publish this class to the "logging.handlers" module so that it can be use
+    """A concurrent log handler that does time-based-rotation instead of size-based.
+    See the docs for TimedRotatingFileHandler."""
+    def __init__(
+        self, filename, when='h', interval=1, backupCount=0,
+        encoding=None, delay=False, utc=False, atTime=None,
+        errors=None, mode='a', debug=False, use_gzip=False,
+        owner=None, chmod=None, umask=None, newline=None, terminator="\n",
+        unicode_error_policy='ignore'
+    ):
+        TimedRotatingFileHandler.__init__(
```

Comment:
you shouldn’t need to call init methods explicitely in a specific order, python’s `mro` ensures that `super()` will resolve these in order, and that multiple inheritance works. https://rhettinger.wordpress.com/2011/05/26/super-considered-super/

Reply:
I think the problem there is the different method signatures (arguments) on TimedRotatingFH vs ConcurrentFHMixin. Your link points out the need for the method signatures to align. While there might be ways to use `**kwargs` creatively here, I tend to prefer explicitly naming all the arguments in each place. Method resolution order wasn't the problem, it's the incompatibility of the method signatures.